lib/actions/lookupObject.js

```diff
@@ -27,14 +27,29 @@ module.exports.getLookupFieldsModel = function getLookupFieldsModel(configuratio
   return metaLoader.getLookupFieldsModelWithTypeOfSearch(configuration.typeOfSearch);
 };
 
+/**
+ * This function will return a metamodel description for a particular object
+ *
+ * @param configuration
+ */
+module.exports.getLinkedObjectsModel = function getLinkedObjectsModel(configuration) {
+  const metaLoader = new MetaLoader(configuration, this);
+  return metaLoader.getLinkedObjectsModel();
+};
+
 /**
  * This function will process Action
  *
  * @param message - contains data for processing
  * @param configuration - contains configuration data for processing
  */
 module.exports.process = async function processAction(message, configuration) {
-  const { allowCriteriaToBeOmitted, allowZeroResults, lookupField } = configuration;
+  const {
+    allowCriteriaToBeOmitted,
+    allowZeroResults,
+    lookupField,
+    linkedObjects,
```

Comment:
There is a way you can extend this line so that `linkedObjects` defaults to `[]` if it is not defined.